README.md

```diff
@@ -37,7 +37,7 @@ Upstream links:
 
 * Retrieve the client configuration with embedded certificates
 
-        docker run --volumes-from $OVPN_DATA --rm kylemanna/openvpn ovpn_getclient CLIENTNAME > CLIENTNAME.ovpn
+        docker run --volumes-from $OVPN_DATA --rm kylemanna/openvpn ovpn_getclient CLIENTNAME combined > CLIENTNAME.ovpn
```

Comment:
I first Idea also was to default to combined to not break the tutorials. But then I thought, that the other feature might not be visible to the user in this way. So I made it explicate. But ok, I will change it back.